src/modules/media/services/media/media.service.ts

```diff
@@ -115,6 +115,24 @@ export class MediaService {
 			.json();
 	}
 
+	public static async countRelated(
+		meemooIdentifiers: string[] = []
+	): Promise<Record<string, number>> {
+		return await ApiService.getApi()
+			.get(
+				stringifyUrl(
+					{
+						url: `${MEDIA_SERVICE_BASE_URL}/related/count`,
+						query: { meemooIdentifiers },
```

Comment:
side lore:
not sure if you took this into account, but concatinating ids can sometimes give issues since the max url length is 2048 chars
lucky for you, the meemoo id is only 10 chars long
so 39 * 10 => 310 chars

But if you would build this endpoint with schema_identifiers which are 96 chars long you would run into issues:
96 * 39 => 3744 chars

Reply:
wanted to keep it `GET` but yeah, long term we should look at an equivalent `POST`